satellite/audit/verifier.go

```diff
@@ -95,7 +95,7 @@ func (verifier *Verifier) Verify(ctx context.Context, path storj.Path, skip map[
 
 	defer func() {
 		// if piece hashes have not been verified for this segment, do not mark nodes as failing audit
-		if !pointer.PieceHashesVerified {
+		if !pointer.PieceHashesVerified && report != nil {
```

Comment:
If you make it return a `Report` instead of `*Report` it's safer.

Reply:
done